.eslintignore

```diff
@@ -0,0 +1 @@
+typings/index.d.ts
```

Comment:
There's [a default eslintignore with kcd-scripts](https://github.com/kentcdodds/kcd-scripts/blob/master/src/config/eslintignore) and once you add an `.eslintignore` file, then the default no longer applies. Please include the files from the default here as well.